SU-SWS/stanford_profile
# D8CORE-1513: Allow site managers and higher to flush caches

# READY FOR REVIEW 

# Summary
- Alters the permissions on the route to provide a flush cache option
- New module to do all this in prep for the move out of the profile

# Needed By (Date)
- ON HOLD.
- Review again in June

# Urgency
- None

# Steps to Test

0. Install a new site with this branch checked out or...
1. Check out this branch
2. Enable the stanford_profile_permissions module
3. Import config and clear caches
4. Log in as user 3 Morgan
5. Hover over the toolbar to see dropdown options an validate that the user can see and execute the flush cache

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
